marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/builtins/steps/mastering/default/manual-merge.mjs

```diff
@@ -1,23 +1,19 @@
-import masteringStepLib from "/data-hub/5/builtins/steps/mastering/default/lib.mjs";
 import hubUtils from "/data-hub/5/impl/hub-utils.mjs";
-
-const merging = require('/com.marklogic.smart-mastering/merging.xqy');
-const collImpl = require('/com.marklogic.smart-mastering/survivorship/merging/collections.xqy');
+import Mergeable from "/data-hub/5/mastering/merging/mergeable.mjs";
+import masteringStepLib from "/data-hub/5/builtins/steps/mastering/default/lib.mjs";
+import {ThresholdDefinition} from "/data-hub/5/mastering/matching/matchable.mjs";
 
 const quickStartRequiredOptionProperty = 'mergeOptions';
 const hubCentralRequiredOptionProperty = 'mergeRules';
 
 function main(content, options) {
   masteringStepLib.checkOptions(null, options, null, [[quickStartRequiredOptionProperty,hubCentralRequiredOptionProperty]]);
-  let uris = [];
-  let mergeOptions = new NodeBuilder().addNode(options).toNode();
-  for (const item of content) {
-    uris.push(item.uri);
-    item.context.collections = collImpl.onArchive({ [item.uri]: Sequence.from(item.context.originalCollections) }, mergeOptions.xpath('(mergeOptions/algorithms/collections|targetCollections)/onArchive'));
-  }
-  let mergedDocument = fn.head(merging.buildMergeModelsByUri(uris, mergeOptions));
+
+  const mergeable = new Mergeable(options);
+  const threshold = new ThresholdDefinition({"action":"merge"}, null); 
   let contentArray = hubUtils.normalizeToArray(content);
-  contentArray.push(mergedDocument['audit-trace']);
+  const uri = threshold.generateActionURI(contentArray);
+  let mergedDocument = {uri: uri, previousUris: contentArray.map(c => c.uri), value: mergeable.buildMergeDocument(contentArray), context: {}}
   let filteredContent = contentArray.filter((item) => item.uri !== mergedDocument.uri);
```

Comment:
I think the test failures on this PR are because collections aren't being applied to the merged document.

You can apply the permissions/collections with `mergeable.applyDocumentContext(mergedDocument, {"action":"merge"})`